sources/System/Console/Trace.h

```diff
@@ -30,7 +30,7 @@ class Trace : public T_Singleton<Trace> {
   Trace::Logger *SetLogger(Trace::Logger &);
 
 protected:
-  static void VLog(const char *category, const char *fmt, const va_list &args);
+  static void VLog(const char *category, const char *fmt, va_list &args);
```

Comment:
I'm not sure why this `const` would be a problem. @shockdesign  are you perhaps using a diff compiler version because you are on a intel macos device?